src/libcore/default.rs

```diff
@@ -115,6 +115,48 @@ pub trait Default: Sized {
     fn default() -> Self;
 }
 
+/// Equivalent to [`Default::default()`] but without repeating the word
+/// "default" twice at the call site.
```

Comment:
A proposal would be to word this in a positive way, i.e describe what this function does well, instead of describing what its alternative does less well, maybe something like this:

Return the default value of a type according to the `Default` trait.

The type to return is inferred from context; this is a shorter but equivalent way of calling `Default::default()`.

Reply:
Used your suggestion.